From 341c33cbd0ebc55f5b5f32ed1ef2b55b471d121f Mon Sep 17 00:00:00 2001
From: Larisa Ileana Grigore <larisa.grigore@nxp.com>
Date: Fri, 22 Jan 2021 03:41:35 -0600
Subject: [PATCH 59/76] s32g_emu: Remove sram initialization

Currently the SRAM initialization causes an exception
so it will be temporary removed.

Issue: ALB-6137
Signed-off-by: Larisa Ileana Grigore <larisa.grigore@nxp.com>
---
 arch/arm/cpu/armv8/s32/lowlevel.S | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/cpu/armv8/s32/lowlevel.S b/arch/arm/cpu/armv8/s32/lowlevel.S
index 6d4f788e73..7792fc82bd 100644
--- a/arch/arm/cpu/armv8/s32/lowlevel.S
+++ b/arch/arm/cpu/armv8/s32/lowlevel.S
@@ -1,7 +1,7 @@
 /* SPDX-License-Identifier:     GPL-2.0+ */
 /*
  * (C) Copyright 2014 Freescale Semiconductor
- * Copyright 2017-2020 NXP
+ * Copyright 2017-2021 NXP
  *
  * Extracted from armv8/start.S
  */
@@ -46,6 +46,7 @@ watchdog_cortexm_disable:
 
 /* Skip SRAM initialization if running with ATF and U-Boot in DDR */
 #if !(defined(CONFIG_S32_ATF_BOOT_FLOW) && defined(CONFIG_S32_SKIP_RELOC))
+#ifndef CONFIG_TARGET_TYPE_S32GEN1_EMULATOR
 sram_init:
 	/* Clear stack region */
 	ldr x0,  =(CONFIG_SYS_INIT_SP_ADDR - CONFIG_SYS_INIT_SP_OFFSET)
@@ -66,9 +67,9 @@ sram_init:
 	sub x1, x1, x0
 
 	bl sram_clr
-
 	/* turn on a53 slave cores from a53 master */
 	/* deassert cores on reset */
+#endif /* CONFIG_TARGET_TYPE_S32GEN1_EMULATOR */
 #endif
 
 start_slave_cores:
-- 
2.17.1

